bdd-generator/src/test/resources/application.yml

```diff
@@ -3,4 +3,41 @@ spring.datasource.url: jdbc:h2:mem:testDB;DB_CLOSE_DELAY=-1
 spring.datasource.username: sa
 spring.datasource.password: sa
 spring.jpa.hibernate.naming.physical-strategy: org.hibernate.boot.model.naming.PhysicalNamingStrategyStandardImpl
-spring.liquibase.change-log: classpath:db/changelog/db.changelog-master.xml
\ No newline at end of file
+spring.liquibase.change-log: classpath:db/changelog/db.changelog-master.xml
+spring:
+  mail:
+    default-encoding: UTF-8
+    host: smtp.gmail.com
+    port: 587
+    username: cucumbervarificator@gmail.com
+    password: cucumberadmin
+    properties:
+      mail:
+        smtp:
+          auth: true
+          starttls:
+            enable: true
+    protocol: smtp
+subject.registration.message: Email verification
```

Comment:
Duplication is not required here. Tests would pass if you leave only:

```
spring.datasource.driver-class-name: org.h2.Driver
spring.datasource.url: jdbc:h2:mem:testDB;DB_CLOSE_DELAY=-1
spring.datasource.username: sa
spring.datasource.password: sa
spring.jpa.hibernate.naming.physical-strategy: org.hibernate.boot.model.naming.PhysicalNamingStrategyStandardImpl
spring.liquibase.change-log: classpath:db/changelog/db.changelog-master.xml
spring:
  mail:
    default-encoding: UTF-8
    host: smtp.gmail.com
    port: 587
    username: test
    password: test
subject.registration.message: Email verification
site.name: https://www.bddgenerator.com
registration.message: registration message
subject.password.message: password message
reset.password.message: password reset message
```

Reply:
Removed duplicates